packages/connect-form/src/components/advanced-connection-options.tsx

```diff
@@ -0,0 +1,61 @@
+import { css } from '@emotion/css';
+import React from 'react';
+import { Accordion, spacing } from '@mongodb-js/compass-components';
+import ConnectionStringUrl from 'mongodb-connection-string-url';
+
+import AdvancedOptionsTabs from './advanced-options-tabs/advanced-options-tabs';
+import {
+  ConnectFormFields,
+  SetConnectionField,
+} from '../hooks/use-connect-form';
+
+const disabledOverlayStyles = css({
+  position: 'absolute',
+  top: 0,
+  // Space around it to ensure added focus borders are covered.
+  bottom: -spacing[1],
+  left: -spacing[1],
+  right: -spacing[1],
+  backgroundColor: 'rgba(255, 255, 255, 0.5)',
+  zIndex: 1,
+  cursor: 'not-allowed',
+});
+
+const connectionTabsContainer = css({
+  position: 'relative',
+});
+
+function AdvancedConnectionOptions({
+  disabled,
+  fields,
+  connectionStringUrl,
+  setConnectionField,
+  setConnectionStringUrl,
+}: {
+  disabled: boolean;
+  fields: ConnectFormFields;
+  connectionStringUrl: ConnectionStringUrl;
+  setConnectionField: SetConnectionField;
+  setConnectionStringUrl: (connectionStringUrl: ConnectionStringUrl) => void;
+}): React.ReactElement {
+  return (
+    <Accordion text="Advanced Connection Options">
+      <div className={connectionTabsContainer}>
+        {disabled && (
+          <div
```

Comment:
We'll be addressing this more thoroughly in a future ticket, we could remove from this one, added it for a bit of temporary ux testing since it's similarly to what it will be like.